client/my-sites/stats/stats-site-overview/index.jsx

```diff
@@ -81,3 +76,20 @@ export default React.createClass( {
 		);
 	}
 } );
+
+export default connect( ( state, ownProps ) => {
+	const { siteId, date, period, siteSlug } = ownProps;
+	const query = {
+		date,
+		period
+	};
+	// It seems not all sites are in the sites/items subtree consistently
```

Comment:
At the moment we only populate selected site on page load and when we switch sites. We do have a QuerySites components, but it's not in use until we're ready to flip everything over.